pkg/kubelet/kubelet_pods.go

```diff
@@ -1744,14 +1744,19 @@ func (kl *Kubelet) convertToAPIContainerStatuses(pod *v1.Pod, podStatus *kubecon
 		// and if the pod itself is being deleted, then the CRI may have removed the container already and for whatever reason the kubelet missed the exit code
 		// (this seems not awesome).  We know at this point that we will not be restarting the container.
 		status := statuses[container.Name]
+
+		if len(podStatus.ContainerStatuses) == 0 && status.State.Terminated != nil {
+			continue
+		}
```

Comment:
To make pod terminating smoothly, the container statuses would be empty array during terminating.  
Any suggestions here?